cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,131 @@
+/// <reference types="Cypress"/>
+
+describe('EngageSphere', () => {
+    beforeEach(() => {
+
+        cy.visit('')
+        
+    })
+
+    it('Valida se H1 esta visivel e alterna tema', () => {
+        cy.contains('h1', 'EngageSphere').should('be.visible')
+        cy.get('#theme-toggle-button').click()
+    })
+
+    it.only('Valida mensagem de saudação', () => {
```

Comment:
```suggestion
    it('Valida mensagem de saudação', () => {
```

Cuidado para não fazer commit de testes com `.only`.

> O mesmo vale para o próximo teste também.